client/cli/go/cmds/volume.go

```diff
@@ -61,14 +61,11 @@ func init() {
 			"\n\t\treplicate: (Default) Distributed-Replica volume."+
 			"\n\t\tdisperse: Distributed-Erasure Coded volume.")
 	volumeCreateCommand.Flags().IntVar(&replica, "replica", 3,
-		"\n\tReplica value for durability type 'replicate'."+
-			"\n\tDefault is 3")
+		"\n\tReplica value for durability type 'replicate'.")
```

Comment:
These changes to the default string should probably be in a separate commit.

Reply:
@phlogistonjohn done separate commit for string dropping

Reply:
added separate commit for string dropping